### PR TITLE
DOC-2498: Pasting plain text with the `mceInsertClipboardContent` command is now a synchronous action, to match TinyMCE core behaviour.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2498

Site: [Staging branch](http://docs-feature-75-doc-2498tiny-11389.staging.tiny.cloud/docs/tinymce/latest/7.5-release-notes/#pasting-plain-text-with-the-mceinsertclipboardcontent-command-is-now-a-synchronous-action-to-match-tinymces-core-behaviour)

Changes:
* improvement for TINY-11389

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/7.5-release-notes.adoc [hunk 1]
```diff
@@ -83,6 +83,23 @@ In previous versions of the tinycomments plugin, the `conversationAuthor` proper
...
+
+==== Pasting plain text with the `mceInsertClipboardContent` command is now a synchronous action, to match {productname}'s core behaviour. 
+
+Previously, the `mceInsertClipboardContent` command in Powerpaste was executed asynchronously, diverging from {productname}'s core behavior and causing compatibility issues with the xref:markdown.adoc[Markdown] plugin.
+
+As a consequence, this asynchronous execution rendered Markdown and Powerpaste entirely incompatible. When Powerpaste was active, the Markdown plugin was effectively disabled, as it relied on synchronous command execution.
+
+{productname} {release-version} addresses this issue. Now, the `mceInsertClipboardContent` command within the Markdown plugin has been updated to be synchronous, aligning it with {productname}'s core functionality.
+
+As a result, the command now executes synchronously, allowing both Powerpaste and the Markdown plugin to operate together seamlessly, restoring full functionality for Markdown users.
```
Comment (HAFRMO) [inline modules/ROOT/pages/7.5-release-notes.adoc:100]: This is not correct. The command as a whole is still async.
If the command is only inserting plaintext ( `editor.execCommand('mceInsertClipboardContent', false, { text: content })` it is a sync operation ( which is also what the Markdown-plugin uses ).
If the user inserts html ( `editor.execCommand('mceInsertClipboardContent', false, { html: content })` it remains async as it performs some actions which requires async operations for html content.
I apologize if my documentation entry was poorly worded to cause this confusion.

**Review by HAFRMO — DISMISSED** [review]

File: modules/ROOT/pages/7.5-release-notes.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/pages/7.5-release-notes.adoc:100]: Thanks for the feedback @HAFRMO - Im confident that @FarzadHayat updated context covers this well.

File: modules/ROOT/pages/7.5-release-notes.adoc [hunk 2]
```diff
@@ -83,6 +83,21 @@ In previous versions of the tinycomments plugin, the `conversationAuthor` proper
...
+
+**PowerPaste** Premium plugin includes the following improvement.
+
+==== Pasting plain text with the `mceInsertClipboardContent` command is now a synchronous action, to match {productname}'s core behaviour. 
+
+Previously in {productname}, when the Powerpaste plugin was enabled, the `mceInsertClipboardContent` command was executed asynchronously. This caused compatibility issues with the xref:markdown.adoc[Markdown] plugin, as the Markdown plugin relied on synchronous command execution. As a consequence, the Markdown plugin was effectively disabled when Powerpaste was active.
+
+{productname} {release-version} addresses this issue. The `mceInsertClipboardContent` command has been updated to be conditionally synchronous when the Powerpaste plugin is enabled. If the content is plaintext (e.g., `editor.execCommand('mceInsertClipboardContent', false, { text: content })`), the command is executed synchronously, aligning it with {productname}'s core functionality. If the content is HTML (e.g., `editor.execCommand('mceInsertClipboardContent', false, { html: content })`), the command remains asynchronous, as it requires async operations for HTML content.
```
Comment (kemister85) [inline modules/ROOT/pages/7.5-release-notes.adoc:96]:
```suggestion
{productname} {release-version} addresses this issue. The `mceInsertClipboardContent` command has been updated to be conditionally synchronous when the Powerpaste plugin is enabled. If the content is plaintext (e.g., `+editor.execCommand('mceInsertClipboardContent', false, { text: content })+`), the command is executed synchronously, aligning it with {productname}'s core functionality. If the content is HTML (e.g., `+editor.execCommand('mceInsertClipboardContent', false, { html: content })+`), the command remains asynchronous, as it requires async operations for HTML content.
```

File: modules/ROOT/pages/7.5-release-notes.adoc [hunk 3]
```diff
@@ -179,6 +179,22 @@ This behavior led to confusion and increased the risk of interacting with obsole
...
+
+{productname} {release-version} addresses this issue. The `mceInsertClipboardContent` command has been updated to be conditionally synchronous when the Powerpaste plugin is enabled. If the content is plaintext (e.g., `+editor.execCommand('mceInsertClipboardContent', false, { text: content })+`), the command is executed synchronously, aligning it with {productname}'s core functionality. If the content is HTML (e.g., `+editor.execCommand('mceInsertClipboardContent', false, { html: content })+`), the command remains asynchronous, as it requires async operations for HTML content.
+
+As a result, the `mceInsertClipboardContent` command now always executes synchronously when pasting plaintext, allowing both Powerpaste and the Markdown plugin to operate together seamlessly, restoring full functionality for Markdown users.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/7.5-release-notes.adoc:194]:
```suggestion
As a result, the `mceInsertClipboardContent` command now always executes synchronously when pasting plaintext, allowing both Powerpaste and Markdown plugins to operate together seamlessly, restoring full functionality for Markdown users.
```

**Review by tiny-ben-tran — APPROVED** [review]
Should there be an update to Markdown doc to flag it is no longer disabled from 7.5 release?

**Comment by kemister85** [comment]
> Should there be an update to Markdown doc to flag it is no longer disabled from 7.5 release?

The only change to the Markdown Plugin was we added a comment to `Commands.ts`, so i think we are ok here.

**Pull request merged**